### PR TITLE
[fix bug 1258324] Allow all mozillians to accept/reject invitations.

### PR DESCRIPTION
**Pull request by akatsoulas**
@johngian r?

### CODE REVIEW

File: mozillians/groups/views.py [hunk 1]
```diff
@@ -166,14 +166,14 @@ def show(request, url, alias_model, template):
...
             show_delete_group_button = is_curator and group.members.all().count() == 1
-            invitation = get_object_or_none(Invite, redeemer=profile, group=group, accepted=False)
-            data.update(invitation=invitation)
 
         else:
             # only show full members, or this user
             memberships = group.groupmembership_set.filter(
                 Q(status=GroupMembership.MEMBER) | Q(userprofile=profile))
 
+        invitation = get_object_or_none(Invite, redeemer=profile, group=group, accepted=False)
```
Comment (johngian) [inline mozillians/groups/views.py]: I think that you should check here if the group that we are trying to render is a ``Group`` instance and not a ``Skill``.

Comment (akatsoulas) [inline mozillians/groups/views.py]: this is inside the if clause that checks for a group instance

Comment (johngian) [inline mozillians/groups/views.py]: :+1:

**Comment by johngian** [comment]
r+

**Pull request merged**